### PR TITLE
Remove the CanastaUtils.php file (remove error message about $wgSMTP)

### PR DESCRIPTION
**Pull request by pastakhov**

### CODE REVIEW

**Comment by yaronkoren** [comment]
@pastakhov - can you explain this change? Isn't it useful for people to know that emails will not be sent?

**Comment by pastakhov** [comment]
This is for the `wikiteq` branch, and the docker image can send emails well if Postfix on the host is configured to accept emails from the docker network. We usually configure postfix on all servers, and the message is confusing for us

**Comment by yaronkoren** [comment]
Oh, I didn't realize this was for the wikiteq branch. Thanks for the explanation.

Better yet, I would think, would be some flag to tell Canasta that mailing does work (outside of $wgSMTP). Or maybe Postfix can just be added to Canasta? I didn't realize this was a possibility.

**Comment by pastakhov** [comment]
It is easier to have one configured Postfix on the host which worries about sending email messages. All Canasta and any other service on the server needs is to send the emails to Postfix on the host. This is the best practice.

**Comment by yaronkoren** [comment]
Thanks for letting me know - I will definitely look into this Postfix thing.

Feel free to do whatever you want with this patch, though I still think the best option is to keep this code, but add another configuration variable for the PHP code to check, or some other way for the code to know that emailing is enabled.

**Review by vedmaka — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -758,7 +758,6 @@ COPY _sources/scripts/*.php $MW_HOME/maintenance/
 COPY _sources/configs/robots.txt $WWW_ROOT/
 COPY _sources/configs/.htaccess $WWW_ROOT/
 COPY _sources/images/favicon.ico $WWW_ROOT/
-COPY _sources/canasta/LocalSettings.php _sources/canasta/CanastaUtils.php _sources/canasta/CanastaDefaultSettings.php $MW_HOME/
```
[the remark this answers is not in the document]
Comment (pastakhov) [reply Dockerfile:761]: Oh, thanks. I missed the CanastaDefaultSettings.php file

[the remark this answers is not in the document]
Comment (pastakhov) [reply Dockerfile:761]: OMG, and I missed the LocalSettings.php and I didn't remove the CanastaUtils.php file...  looks like I need to drink a little less

**Review by vedmaka — APPROVED** [review]

**Comment by jeffw16** [comment]
By the way, I would highly recommend adding back the `CanastaUtils.php` file, even if it's empty. I understand that none of the contents were applicable to Taqasta, and I agree with removing those lines of code. However, removing files will make the Canasta–Taqasta divide even greater.

**Comment by jeffw16** [comment]
We always reserve the right to add stuff to CanastaUtils.php, so I don't think we should be removing the file.

**Pull request merged**